docs/index.html

```diff
@@ -139,7 +148,7 @@ <h3>SNARVs: Succinct Non-Interactive Arguments of Voting &ndash; protocols for c
 
 </div>
 
-</a><a href='https://forum.aragon.org/t/blind-ovote-ovote-with-blind-signatures-for-voter-anonymity/3881'><div class="row postThumb">
+</a><a href='/.html'><div class="row postThumb">
```

Comment:
This seems to be due an usage of an old blogo version (which does not support listing articles from outside the blog). It will not be much problem as the github action will re-generate the html, but for future local builds I would recommend using the latest blogo version from: https://github.com/arnaucube/blogo/releases/tag/v0_20221124